curator-framework/src/test/java/org/apache/curator/framework/imps/TestFramework.java

```diff
@@ -755,19 +754,6 @@ public void testNullNamespace() {
         CloseableUtils.closeQuietly(client);
     }
 
-    @Test
-    public void testNoPartialConstruction() {
```

Comment:
The original behavior test is covered by testNullNamespace - https://github.com/apache/curator/pull/475

We don't need this detailed test.